pandapipes/io/convert_format.py

```diff
@@ -42,6 +42,13 @@ def _rename_columns(net):
             else:
                 net['controller'].drop('controller', inplace=True, axis=1)
         net["controller"].rename(columns={"controller": "object"}, inplace=True)
+    for cp_tbl in ["circ_pump_mass", "circ_pump_pressure"]:
+        if cp_tbl in net:
+            for old_col, new_col in [("from_junction", "flow_junction"),
+                                     ("to_junction", "return_junction")]:
+                if old_col in net[cp_tbl] and new_col not in net[cp_tbl]:
+                    net[cp_tbl][new_col] = net[cp_tbl][old_col]
+                    net[cp_tbl].drop(old_col, inplace=True, axis=1)
```

Comment:
why not using the rename functionality of pandas? Maybe, it would make sense to pass the component here as well. By that, if the name changes again, you do not need to adapt this as well.

Reply:
Did that in the new commit.